package.json

```diff
@@ -97,6 +97,9 @@
     "typescript": "~5.2.0",
     "verdaccio": "^5.24.1"
   },
+  "overrides": {
+    "fast-xml-parser": "^4.4.1"
+  },
```

Comment:
I'd rather wait for sdk to ship patch for this.
This override won't affect how our packages are consumed from npm.